cmake/ProtobufGenCpp.cmake

```diff
@@ -0,0 +1,97 @@
+# Shipped from CMake 3.9.6, with small modification
```

Comment:
Please update Acknowledgements.txt accordingly (as we takes pieces from CMake).

Reply:
To make sure, if it is ok to add the Copyright content of CMake to ACK

Reply:
Anything I need to do ?